### ISSUE TITLE
Can't create or mount jottacloud remote.

### ISSUE THREAD

**Issue by Daniel41550** [rclone/rclone#3434]
#### What is the problem you are having with rclone?
When try to create or mount a jottacloud remote, it gives this error:
Error getting devices: HTTP error 401 (401 Unauthorized) returned body: "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n\n<error>\n  <code>401</code>\n  <message>org.springframework.security.authentication.AuthenticationCredentialsNotFoundException: Token mismatch!</message>\n  <reason>Unauthorized</reason>\n  <cause></cause>\n  <hostname></hostname>\n  <x-id>311257289867</x-id>\n</error>\n"

#### What is your rclone version (output from `rclone version`)
v1.48.0-157-ga1263e70-beta

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)
Windows 10, 64 bit

####  Which cloud storage system are you using? (eg Google Drive)
Jottacloud

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
rclone config (create)
rclone mount

#### A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)
```
$ rclone config -vv
2019/08/12 11:43:12 DEBUG : rclone: Version "v1.48.0-157-ga1263e70-beta" starting with parameters ["rclone" "config" "-vv"]
2019/08/12 11:43:12 DEBUG : Using config file from "C:\\Users\\Daniel\\.config\\rclone\\rclone.conf"
No remotes found - make a new one
n) New remote
s) Set configuration password
q) Quit config
n/s/q> n
name> jo
Type of storage to configure.
Enter a string value. Press Enter for the default ("").
Choose a number from below, or type in your own value
 1 / 1Fichier
   \ "fichier"
 2 / A stackable unification remote, which can appear to merge the contents of several remotes
   \ "union"
 3 / Alias for an existing remote
   \ "alias"
 4 / Amazon Drive
   \ "amazon cloud drive"
 5 / Amazon S3 Compliant Storage Provider (AWS, Alibaba, Ceph, Digital Ocean, Dreamhost, IBM COS, Minio, etc)
   \ "s3"
 6 / Backblaze B2
   \ "b2"
 7 / Box
   \ "box"
 8 / Cache a remote
   \ "cache"
 9 / Dropbox
   \ "dropbox"
10 / Encrypt/Decrypt a remote
   \ "crypt"
11 / FTP Connection
   \ "ftp"
12 / Google Cloud Storage (this is not Google Drive)
   \ "google cloud storage"
13 / Google Drive
   \ "drive"
14 / Google Photos
   \ "google photos"
15 / Hubic
   \ "hubic"
16 / JottaCloud
   \ "jottacloud"
17 / Koofr
   \ "koofr"
18 / Local Disk
   \ "local"
19 / Mega
   \ "mega"
20 / Microsoft Azure Blob Storage
   \ "azureblob"
21 / Microsoft OneDrive
   \ "onedrive"
22 / OpenDrive
   \ "opendrive"
23 / Openstack Swift (Rackspace Cloud Files, Memset Memstore, OVH)
   \ "swift"
24 / Pcloud
   \ "pcloud"
25 / QingCloud Object Storage
   \ "qingstor"
26 / SSH/SFTP Connection
   \ "sftp"
27 / Webdav
   \ "webdav"
28 / Yandex Disk
   \ "yandex"
29 / http Connection
   \ "http"
30 / premiumize.me
   \ "premiumizeme"
Storage> 16
** See help for jottacloud backend at: https://rclone.org/jottacloud/ **

User Name:
Enter a string value. Press Enter for the default ("").
user> gmail.com
Edit advanced config? (y/n)
y) Yes
n) No
y/n> n
Remote config

Do you want to create a machine specific API key?

Rclone has it's own Jottacloud API KEY which works fine as long as one only uses rclone on a single machine. When you want to use rclone with this account on more than one machine it's recommended to create a machine specific API key. These keys can NOT be shared between machines.

y) Yes
n) No
y/n> y
2019/08/12 11:43:35 DEBUG : Trying to register device 'rclone-FJvRZ09U8AEUQPt8vKUNE'
2019/08/12 11:43:35 DEBUG : Saving config "client_id" = "kp5b38et0m2q0g96mf36st4ibm" in section "jo" of the config file
2019/08/12 11:43:35 DEBUG : Saving config "client_secret" = "_m0HvdY-aUCTrlOqk04UCBBlSSStI4oteWPi9BZM4gz2Z_K0AJ6G5wK1" in section "jo" of the config file
2019/08/12 11:43:35 DEBUG : Got clientID 'kp5b38et0m2q0g96mf36st4ibm' and clientSecret 'o1a1mb9cieunov8utji8fpucff'
Your Jottacloud password is only required during setup and will not be stored.
password:
2019/08/12 11:43:45 DEBUG : jo: Added new token to config, still needs to be saved

Do you want to use a non standard device/mountpoint e.g. for accessing files uploaded using the official Jottacloud client?

y) Yes
n) No
y/n> y
2019/08/12 11:43:47 Error getting devices: HTTP error 401 (401 Unauthorized) returned body: "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n\n<error>\n  <code>401</code>\n  <message>org.springframework.security.authentication.AuthenticationCredentialsNotFoundException: Token mismatch!</message>\n  <reason>Unauthorized</reason>\n  <cause></cause>\n  <hostname></hostname>\n  <x-id>683162424100</x-id>\n</error>\n"
```

**Comment by Tristar7** [comment]
Just getting this as well, can't access Jottacloud through rclone at all.

**Comment by buengese** [comment]
Yep looks like jottacloud changed their api yet again. Will look into this tomorrow.

**Comment by ncw** [comment]
> Yep looks like jottacloud changed their api yet again. Will look into this tomorrow.

Thank you :-)

I've scheduled the next release for 24 August - it would be great if we could make it work for then.  If not then I can push the release (I've already pushed it 4 weeks though!)

**Comment by olihey** [comment]
It works for me when I use just my Jottacloud username "olihey"
```
Storage> 14
** See help for jottacloud backend at: https://rclone.org/jottacloud/ **

User Name:
Enter a string value. Press Enter for the default ("").
user> olihey
Edit advanced config? (y/n)
y) Yes
n) No
y/n> n
Remote config

Do you want to create a machine specific API key?

Rclone has it's own Jottacloud API KEY which works fine as long as one only uses rclone on a single machine. When you want to use rclone with this account on more than one machine it's recommended to create a machine specific API key. These keys can NOT be shared between machines.

y) Yes
n) No
y/n> y
Your Jottacloud password is only required during setup and will not be stored.
password:

Do you want to use a non standard device/mountpoint e.g. for accessing files uploaded using the official Jottacloud client?

y) Yes
n) No
y/n> Y
Please select the device to use. Normally this will be Jotta
Choose a number from below, or type in an existing value
 1 > Jotta
 2 > OLIWIN10
 3 > WINAS
Devices> 1
Please select the mountpoint to user. Normally this will be Archive
Choose a number from below, or type in an existing value
 1 > Archive
 2 > Photos
 3 > Shared
 4 > Sync
```
If I use the FULL email it fails
```
User Name:
Enter a string value. Press Enter for the default ("").
user> olihey@googlemail.com
Edit advanced config? (y/n)
y) Yes
n) No
y/n> n
Remote config

Do you want to create a machine specific API key?

Rclone has it's own Jottacloud API KEY which works fine as long as one only uses rclone on a single machine. When you want to use rclone with this account on more than one machine it's recommended to create a machine specific API key. These keys can NOT be shared between machines.

y) Yes
n) No
y/n> n
Your Jottacloud password is only required during setup and will not be stored.
password:

Do you want to use a non standard device/mountpoint e.g. for accessing files uploaded using the official Jottacloud client?

y) Yes
n) No
y/n> y
2019/08/13 12:55:18 Error getting devices: HTTP error 401 (401 Unauthorized) returned body: "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n\n<error>\n  <code>401</code>\n  <message>org.springframework.security.authentication.AuthenticationCredentialsNotFoundException: Token mismatch!</message>\n  <reason>Unauthorized</reason>\n  <cause></cause>\n  <hostname></hostname>\n  <x-id>999375902116</x-id>\n</error>\n"
```
Not a solution but maybe a workaround

**Comment by Tristar7** [comment]
>It works for me when I use just my Jottacloud username "olihey"

Is there any way to set up a username? I cannot find such an option (maybe a legacy thing?)

**Comment by olihey** [comment]
Well, I couldn't find the "username" setting either. So yes, I might be a legacy thing. Sorry

**Comment by buengese** [comment]
Fix available in #3436
